### PR TITLE
[ENH] Redefinition of environments

### PR DESCRIPTION
**Pull request by clement-dufaure**
Rewrite environments names with backward compatibility
Addition on new V2 integration and production environment
https://github.com/InseeFr/Keycloak-FranceConnect/issues/109

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ $ mvn clean install wildfly:deploy
 
 ## How to use it
 
+### Environments
+
+Up until version 6.2.0, the extension provided the so-called V1 and V2 environments, which correspond respectively to the historical FranceConnect "standard" and the FranceConnect Plus.
+
+With the addition of the FranceConnect standard V2 and to clear up any ambiguities, the configurations are now named `nSTANDARD_V2` and `PLUS_V2`, while the previous ones will remain available for a few versions to allow for migration. Since the standard V1 offering will be discontinued in 2025, it will not be included in the new naming conventions.
```
Comment (lme-atolcd) [inline README.en.md:64]: typo: `nSTANDARD_V2` → `STANDARD_V2`

**Review by lme-atolcd — COMMENTED** [review]
LGTM

**Comment by clement-dufaure** [comment]
Urls ok, but v2 probably needs more additions like parmeter prompt (in error with my new configuration FC v2 => waiting for FC support response before open issue)

**Comment by lme-atolcd** [comment]
Tested keycloak-franceconnect 7.0.0 with Keycloak 25.0.6: I was able to login and logout using the `INTEGRATION_STANDARD_V2` environment. Thanks for your work.

I confirm that no `prompt` parameter is added when redirecting the user to the authorize endpoint (https://fcp-low.integ01.dev-franceconnect.fr/api/v2/authorize), but the auth process on the FranceConnect side worked as expected (no error).
I guess that the `prompt` parameter is not (yet ?) required even if this parameter is tagged as required in the documentation (https://docs.partenaires.franceconnect.gouv.fr/fs/migration/fs-migration-diff-v1-v2/#parametres-d-appel-du-endpoint-authorize)

**Pull request merged**